pkg/server/tls/authority/authority.go

```diff
@@ -276,53 +301,42 @@ func (d *DynamicAuthority) notifyWatches(newCertData, newPrivateKeyData []byte)
 			}
 		}
 	}()
-
-	func() {
-		d.signMutex.Lock()
-		defer d.signMutex.Unlock()
-		d.currentCertData = newCertData
-		d.currentPrivateKeyData = newPrivateKeyData
-	}()
 }
 
 // caRequiresRegeneration will check data in a Secret resource and return true
 // if the CA needs to be regenerated for any reason.
-func (d *DynamicAuthority) caRequiresRegeneration(s *corev1.Secret) bool {
```

Comment:
The differences between log `Info` and `Error` calls don't make sense in this function right now.
If we work with two types, all cases should be errors except for the `"Root CA certificate is nearing expiry."` case which would be an Info log.
However, I think making all of these cases info works too.

Reply:
I agree, but I wonder if we should log at Info or Debug level. Your comment suggests Info, but the code Debug. 😉

Reply:
Will fix this.